elastic-search-logger.js

```diff
@@ -4,21 +4,52 @@ module.exports = function (RED) {
   function LogElasticLoggerNode(config) {
     let winston = require('winston');
     let winstonElasticSearch = require('winston-elasticsearch');
-    let os = require("os");
-    this.hostname = os.hostname();
 
     RED.nodes.createNode(this, config);
     this.logger = null;
     let transports = [];
 
     // Elastic settings
-    let url = config.url;
-    let user = this.credentials.username || '';
-    let password = this.credentials.password || '';
-    let index = this.credentials.index || '';
-    if (index == '') {
-      this.error("Elastic search index is not set");
+    let url = "";
+    if(config.urlType == "global"){
+        url = this.context().global.get(config.url);
+    } else {
+        url = config.url;
```

Comment:
Es gibt von Node-RED eine eigene Funktion um TypedInputs auszuwerten:
```suggestion
    let url = RED.util.evaluateNodeProperty(n.url, n.urlType, node);
```